bin/benchmark.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+java -cp "$(cd `dirname $0`/..; pwd)"'/tools/benchmark-cli/build/libs/benchmark-cli.jar:*' \
```

Comment:
Over [here](https://github.com/elastic/logstash/blob/master/tools/benchmark-cli/build.gradle#L55) it seems that you are building using a shadow (fat) jar, but here are using the classpath ?  should this be `java -jar benchmark-cli-shadow.jar` (or the like).